src/hooks/init.ts

```diff
@@ -1,6 +1,8 @@
 import {ux, Interfaces} from '@oclif/core'
-import spawn from 'cross-spawn'
-import * as fs from 'fs-extra'
+import {spawn} from 'child_process'
```

Comment:
Have you tested this on windows? I tried to use `spawn` inside of `plugins install` instead of `fork` and ran into a host of issues on windows. I ended up abandoning the effort and left `fork`.

Reply:
no, I hadn't tried it on windows.  good QA catch!